Core/src/Visualization/GeometryView3D.cpp

```diff
@@ -103,8 +109,8 @@ void Acts::GeometryView3D::drawSurfaceArray(
     const GeometryContext& gctx, const Transform3& transform,
     const ViewConfig& sensitiveConfig, const ViewConfig& passiveConfig,
     const ViewConfig& gridConfig, const std::string& _outputDir) {
-  std::string outputDir =
-      _outputDir == "." ? getWorkingDirectory() : _outputDir;
+  const std::string outputDir =
+      _outputDir == "." ? std::filesystem::current_path().string() : _outputDir;
```

Comment:
Question: Is there a reason why this cannot be a `std::filesystem::path` instead of a string? Generally, I'd advise staying in the `std::filesystem` ecosystem as much as possible.